test/controllers/update/update.beneficial.owner.statements.controller.spec.ts

```diff
@@ -70,7 +69,6 @@ describe("BENEFICIAL OWNER STATEMENTS controller", () => {
       expect(resp.text).toContain(config.LANDING_PAGE_URL);
       expect(resp.text).not.toContain(PAGE_TITLE_ERROR);
       expect(resp.text).toContain(BeneficialOwnersStatementType.ALL_IDENTIFIED_ALL_DETAILS);
-      expect(resp.text).toContain(SAVE_AND_CONTINUE_BUTTON_TEXT);
```

Comment:
The text will depend on the save and continue feature flag for update. You could mock isActiveFeature() or here do:
toContain(config.FEATURE_FLAG_ENABLE_UPDATE_SAVE_AND_RESUME ? SAVE_AND_CONTIUE_BUTTON_TEXT : CONTINUE_BUTTON_TEXT);